scripts/docker-entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ -z "${SOURCECRED_GITHUB_TOKEN}" ]; then
+    echo "Export SOURCECRED_GITHUB_TOKEN with the container."
+    echo "docker run --env SOURCECRED_GITHUB_TOKEN=$(cat .token) -p 8080:8080 sourcecred <repository>"
+    exit 1;
+fi
+
+# serves on 8080 in docker container, should be bound to host
+node /code/bin/sourcecred.js load "${@}"
+yarn start --host 0.0.0.0
```

Comment:
While this is great for development, I think we should definitely move away from this when publishing an image. The underlying webpack dev-server is as the name suggest intended for development with hot-reloading of files and whatnot. I wouldn't dare run it (as root :scream:) on a public facing image by default.

Until we have a better CLI (discussed in #1232), perhaps (for production use) we should encourage users to use this image to build static files for example for gh-pages or as part of a multi-stage build into nginx?

Reply:
That's the only way to expose it to the host, it's still only intended for development. I will add a note to the README to exemplify this... but if we don't set that hostname it's not viewable from the host.

Reply:
Yes the best use case would be to generate those beautiful svg's you showed me - for now I was just starting with the basics (as described in this readme) because the widgets are separate, no?

Reply:
> That's the only way to expose it to the host, it's still only intended for development. I will add a note to the README to exemplify this... but if we don't set that hostname it's not viewable from the host.

Oh I don't mean the `0.0.0.0` host. This is pretty common for docker containers and have no issues with that. I mean `yarn start` is handled by webpack's dev-server and has a lot of features unsafe for production (and a history of having vulnerabilities bubbling up from large dependency list). I would rather this is part of local development only and not the image's default.

For example, I've used `docker-compose` a lot as a way to describe development commands. While the Dockerfile image defaults to production commands.

Reply:
I was just following instructions in the README, I have no way to know what "production" sourcecred looks like, but I'm happy to adjust if you tell me :)